roles/servicetelemetry/defaults/main.yml

```diff
@@ -36,4 +37,7 @@ graphing_spec_config_auth_disable_signout_menu: false
 
 is_k8s: false
 is_openshift: false
+
+# snmp traps values
+snmp_trap_host: "192.168.24.254"
```

Comment:
This seems wrong to me... I don't think you should be specifying the host IP since it'll be dynamic in the environment, no?

Reply:
Oh, this is probably the remote host to connect to I guess? I would think exposing this value in the CRD is probably appropriate.